server/database/config/build/fakeData.sql

```diff
@@ -0,0 +1,499 @@
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-2',
+    'do you love me?',
+    'true_false',
+    '{"answer": true, "options": [true, false]}'
+  ),
+  (
+    'quiz-2',
+    'do you love me?',
+    'true_false',
+    '{"answer": false, "options": [true, false]}'
+  ),
+  (
+    'quiz-2',
+    'do you love me?',
+    'true_false',
+    '{"answer": true, "options": [true, false]}'
+  ),
+  (
+    'quiz-2',
+    'do you love me?',
+    'true_false',
+    '{"answer": false, "options": [true, false]}'
+  ),
+  (
+    'quiz-2',
+    'whats my name',
+    'short_answer',
+    '{"answer": "Zaher", "options": []}'
+  ),
+  (
+    'quiz-2',
+    'whats my name',
+    'short_answer',
+    '{"answer": "Zaher", "options": []}'
+  ),
+  (
+    'quiz-2',
+    'whats my name',
+    'short_answer',
+    '{"answer": "Zaher", "options": []}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  ),
+  (
+    'quiz-3',
+    'What is the capital of Egypt?',
+    'mcq',
+    '{"answer": "Cairo", "options": ["Cairo", "Alexandria", "Luxor", "Aswan"]}'
+  );
+INSERT INTO scores (quiz_id, student_id, student_score)
+VALUES ('quiz-1', 'user-1', '10'),
```

Comment:
This needs to be fixed
```suggestion
VALUES ('quiz-1', 1, '10'),
```